### PR TITLE
[CAL-1094] Embed modal / Inline - UI/Layout/Spacing issues

### LINKED ISSUE #7155: [CAL-1094] Embed modal / Inline - UI/Layout/Spacing issues
Current

Should be

[View in Figma ](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=22%3A38140&t=AQX9GWSFsCzlRrEy-1)

**Note:** We might not need to touch the preview part. Mostly check spacing & components in the left panel, and ensure the modal doesn't touch the edges of the screen.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1094](https://linear.app/calcom/issue/CAL-1094/inline-embed-modal-uilayoutspacing-issues)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## What does this PR do?

Fixes #7155 

## Demo
Screenshot 2023-02-24 at 3 14 16 PM

**Environment**: Staging(main branch) / Production

### CODE REVIEW

File: apps/web/components/Embed.tsx
Comment (CarinaWolli) [inline apps/web/components/Embed.tsx:838]:
```suggestion
    <DialogContent size="lg" className="rounded-lg p-0.5 enableOverflow" type="creation">
```

**Review by CarinaWolli — COMMENTED** [review]
Only left one small comment, other than that looks good to me 🙏

File: apps/web/components/Embed.tsx
Comment (gitstart-calcom) [reply apps/web/components/Embed.tsx]: Thank you for the review @CarinaWolli. You probably meant this?
```suggestion
    <DialogContent size="lg" className="rounded-lg p-0.5" enableOverflow type="creation">
```

**Review by CarinaWolli — APPROVED** [review]
Looking good to me, thank you 🙏

**Comment by gitstart-calcom** [comment]
HI @CarinaWolli! #7156 is blocked by this PR because both are changes to the same component. Can we please get this PR merged soon?

**Comment by CarinaWolli** [comment]
need an approval from @calcom/ui

File: apps/web/components/Embed.tsx
Comment (hariombalhara) [inline apps/web/components/Embed.tsx:711]: Why is it not being used as a Component? A component name starts with capital letter

File: apps/web/components/Embed.tsx
Comment (hariombalhara) [inline apps/web/components/Embed.tsx:712]:
```suggestion
    <Label >
```

**Comment by hariombalhara** [comment]
@gitstart-calcom 
Now:
Screenshot 2023-03-09 at 1 22 39 PM
The embed is showing up in Mobile mode now there is no way to view desktop mode.

Production:
Screenshot 2023-03-09 at 1 24 13 PM


We need to have the width at least the bare minimum that is required to render in desktop mode

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Left some comments

**Comment by gitstart-calcom** [comment]
Hi @hariombalhara, PR has been updated.

With current changes:
Screenshot 2023-03-20 at 4 54 41 PM

File: packages/ui/components/form/color-picker/colorpicker.tsx
Comment (hariombalhara) [inline packages/ui/components/form/color-picker/colorpicker.tsx:36]: Why do we need this?

File: apps/web/components/ui/form/Select.tsx
Comment (hariombalhara) [inline apps/web/components/ui/form/Select.tsx:113]: @sean-brydon Right now there are many select, so I guess it would go everywhere.

Also, @gitstart-calcom can you confirm this change is supposed to apply to all selects as per the design.

File: packages/ui/components/form/color-picker/colorpicker.tsx
Comment (gitstart-calcom) [reply packages/ui/components/form/color-picker/colorpicker.tsx:36]: We needed it for the color picker to work before but it does not seem to be needed now. Will revert the change

File: apps/web/components/ui/form/Select.tsx
Comment (gitstart-calcom) [reply apps/web/components/ui/form/Select.tsx]: Removed the styling to keep it consistent with other dropdown styles

**Comment by gitstart-calcom** [comment]
Hi @hariombalhara, we have made the requested changes. Please review the PR again 🙏

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**
